src/options.c

```diff
@@ -20,6 +20,7 @@ static const char *usage_str =
     " -0, --read-null          Read input delimited by ASCII NUL characters\n"
     " -j, --workers NUM        Use NUM workers for searching. (default is # of CPUs)\n"
     " -i, --show-info          Show selection info line\n"
+		" -x, --exec=PROG          Execute prog on selection. Don't exit.\n"     
```

Comment:
The indentation is different here, than in the surrounding lines

Reply:
Sorry, ;) I literally didn't see it. Fixed it, and trying to push it again.